README.md

```diff
@@ -12,12 +13,13 @@ This program is aimed at short startup time, with all generally used features.
 This is a evince or qpdfview alternative for linux users.
 
 ### Installation
-To Install this program open terminal inside gospel-pdf-viewer-master directory.  
+To Install this program open terminal inside gospel-pdf-viewer directory.  
 First compile UI and Resources file  
 `cd files`  
 `./compile_ui`  
-`./compile_rc`  
+`./compile_rc`
```

Comment:
@dammhannes dont remove trailing two spaces. it is required to denote end of line in a markdown file

Reply:
this is a copy-paste mistake on my side. Please ignore.

-> so essentially my pull request boils down to add pyqt5-dev-tools to the dependencies

Reply:
OK